### DISCUSSION TITLE
scoop update fails because of missing cmd.exe

### DISCUSSION THREAD

**Discussion by bsszds0809** [ScoopInstaller/Scoop#5479] [Help and Questions]
When I run scoop update I get the following error:

PS C:\Users\84987> scoop update
Updating Scoop...
cmd.exe: D:\Scoop\apps\scoop\current\lib\core.ps1:137
Line |
 137 |      cmd.exe /d /c $cmd
     |      ~~~~~~~
     | The term 'cmd.exe' is not recognized as a name of a cmdlet, function, script file, or executable program. Check
     | the spelling of the name, or if a path was included, verify that the path is correct and try again.

System details
Windows version: 11

OS architecture: 64bit

PowerShell version: 7.3.3.0

My PATH now:

PS C:\Users\84987> $env:PATH.Split(';')
C:\Program Files\WindowsApps\Microsoft.PowerShell_7.3.3.0_x64__8wekyb3d8bbwe
C:\WINDOWS
C:\WINDOWS\System32\Wbem
C:\WINDOWS\System32\WindowsPowerShell\v1.0\
C:\WINDOWS\System32\OpenSSH\
D:\JDK\bin
D:\GlobalScoop\shims
D:\Scoop\apps\mysql\current\bin
D:\GlobalScoop\apps\git\current\bin
D:\GlobalScoop\apps\python\current
D:\GlobalScoop\apps\gradle\current\bin
D:\GlobalScoop\apps\erlang\current\bin
D:\GlobalScoop\apps\maven\current\bin
D:\GlobalScoop\apps\nodejs-lts\current
D:\GlobalScoop\apps\nodejs-lts\current\bin
D:\GlobalScoop\apps\python\current\Scripts
D:\GlobalScoop\apps\nodejs-lts\current\node_global
D:\GlobalScoop\apps\nodejs-lts\current\node_global
D:\Scoop\shims
C:\Users\84987\AppData\Local\Microsoft\WindowsApps
C:\Users\84987\AppData\Local\gitkraken\bin
C:\Users\84987\AppData\Local\JetBrains\Toolbox\scripts

**Comment by HUMORCE** [comment]
```
C:\Windows\System32
```

Add this path to $env:PATH
